index.html

```diff
@@ -199,6 +199,82 @@
             #spinner {
                 padding-right: 5px;
             }
+            .modal {
+                display: none;
+                position: fixed;
+                z-index: 1;
+                padding-top: 100px;
+                left: 0;
+                top: 0;
+                width: 100%;
+                height: 100%;
+                overflow: auto;
+                background-color: rgb(0,0,0);
+                background-color: rgba(0,0,0,0.4);
+            }
+            .modal-content {
+                display: flex;
+                background-color: #fefefe;
+                margin: auto;
+                flex-direction: column;
+                align-items: flex-start;
+                width: 30%;
+                box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2),0 6px 20px 0 rgba(0,0,0,0.19);
+                -webkit-animation-name: animatetop;
+                -webkit-animation-duration: 0.4s;
+                animation-name: animatetop;
+                animation-duration: 0.4s
+            }
+            @-webkit-keyframes animatetop {
+                from {top:-300px; opacity:0}
+                to {top:0; opacity:1}
+            }
```

Comment:
It was some example code I found. I should of also sighted where I got it from/author. 

That said, you're suggestion is correct. I'll clean this up.